spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -76,13 +76,13 @@
     },
     "binaries_linux": {
         "mirrors": [
-            "https://osf.io/cs6zt/?action=download",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220512/spinalcordtoolbox-binaries_linux.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
     "binaries_osx": {
         "mirrors": [
-            "https://osf.io/874cy?action=download",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220512/spinalcordtoolbox-binaries_osx.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
```

Comment:
Note: These two updated links currently include only binaries built and packaged for Linux/OSX.

But, since this PR was first opened, we now have Windows binaries to consider. So, I'd also like to try automating the build process for Windows, too, if I can.

Reply:
Possibly fixed in [fe5276b](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/2668/commits/fe5276b17a841a35fc2dc4efe59cb9bc50405140).